src/index.js

```diff
@@ -242,4 +233,7 @@ class CID {
 
 CID.codecs = codecs
 
-module.exports = CID
+module.exports = withIs(CID, {
+  className: 'CID',
+  symbolName: 'cids/CID'
```

Comment:
There's a convention of naming these "@[github org]/[repo name]/[lowercase class name]"...I don't know the reason for this but it might be worth following...

https://github.com/ipld/js-ipld-dag-pb/blob/981cb9fead5a4cd104c012db9db1fc5241eec30c/src/dag-node/index.js#L80
https://github.com/multiformats/js-multiaddr/blob/fd4ff924b2223ef356a6ee893ae4d0b6f5552acd/src/index.js#L49

Reply:
oh interesting, I had assumed it was 1-to-1 with the npm naming convention and those were npm orgs :)